x-pack/plugins/cases/server/routes/api/configure/get_configure.ts

```diff
@@ -6,9 +6,9 @@
  */
 
 import { CASE_CONFIGURE_URL } from '../../../../common/constants';
-import type { GetConfigurationFindRequest } from '../../../../common/api';
 import { createCaseError } from '../../../common/error';
 import { createCasesRoute } from '../create_cases_route';
+import type { configureApiV1 } from '../../../../common/types/api';
```

Comment:
This is probably out of scope for this PR but when we start using the versioned router this file will likely need a version too right?

Reply:
Which file do you mean? `x-pack/plugins/cases/server/routes/api/configure/get_configure.ts`?

Reply:
I was thinking the same but I still need to read a bit more about the versioned router implementation.

If `typeV1` is used here, that makes this `get_configure_ v1` (The same applies to `patch_configure` and `post_configure`also changed in this PR). How do we distinguish these from the same routes using other versions of the API, v2, 3, 4?

Something like
```
routes
    api
      configure
          v1
              get_configure.ts
              ...
          v2
              ...
```
?

Reply:
If it is about the routes, I would say because the route code is small and the breaking changes should be minimal it is ok to have all versions in the same file. If it grows we can think of ways to split it. Not something we should change at the moment.

Reply:
Hmm maybe if it's not too hard, that way we have a complete example going forward?